war/postcss.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  parser: "postcss-less",
+  plugins: [
+    require('autoprefixer'),
+    require('postcss-custom-properties')
```

Comment:
Does the plugin order matter? I'm thinking if there can be a scenario where the CSS variables fallback is applied for a main CSS rule but not for the vendor prefixes.

Reply:
isn't the postcss-custom-properties applied second so it'll be applied on top of the prefixed?

Reply:
No idea, that's why I was asking.

Reply:
I think this is okay